Formula/jobber.rb

```diff
@@ -0,0 +1,59 @@
+class Jobber < Formula
+  desc "Alternative to cron, with better status-reporting and error-handling"
+  homepage "https://dshearer.github.io/jobber/"
+  url "https://github.com/dshearer/jobber/archive/v1.4.3.tar.gz"
+  sha256 "14efd9e088e4b3e1e4018fb8cd3161cc43da7941261b31c3e90d630545b305db"
+  head "https://github.com/dshearer/jobber.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "./configure", "--prefix=#{prefix}", "--libexecdir=#{libexec}", "--sysconfdir=#{etc}",
+      "--localstatedir=#{var}"
+    system "make", "install"
+  end
+
+  plist_options :startup => true
+
+  def plist
+    <<~EOS
+      <?xml version="1.0" encoding="UTF-8"?>
+      <!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+      <plist version="1.0">
+        <dict>
+          <key>Label</key>
+          <string>#{plist_name}</string>
+          <key>Program</key>
+          <string>#{libexec}/jobbermaster</string>
```

Comment:
Why is the binary not in bin?

Reply:
It's a daemon. It shouldn't be on the path.

Reply:
There's a lot of those that are in the bin directory (think servers). What would be the problem with having it there?

Reply:
Jobber has three binaries: jobber (the CLI), jobbermaster (a daemon), and jobberrunner (another daemon)

I thought I read that the convention on Unix is to put the binaries that the user isn’t supposed to invoke directly in libexec.

The benefit I see is that, if the user uses tab completion on “job” or something the daemon binaries won’t show up (which might be confusing).